python_ta/__init__.py

```diff
@@ -153,8 +153,35 @@ def _check(module_name='', reporter=ColorReporter, number_of_messages=5, level='
         print('No checks run. Input to check, `{}`, has invalid type, must be a list of strings.\n'.format(module_name))
         return
 
-    current_reporter = reporter(number_of_messages)
-    linter = _load_pylint_plugins(current_reporter, local_config, pep8)
+    # see 'type' in pylint/config.py `VALIDATORS` dict.
+    new_options = (
```

Comment:
This is good stuff, and it seems like it could be entirely pulled out into a helper function that returns a linter.